rules/linux/defense_evasion_log_files_deleted.toml

```
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/11/03"
+maturity = "production"
+updated_date = "2020/11/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies deletion of Linux sensitive system logs. This may indicate an attempt to evade detection or destroy forensic
+evidence on a system.
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Linux System Log Files Deleted"
```

```suggestion
name = "System Log File Deletion"
```
we may not need to specify `Linux` in the name, as we don't specify in the other Linux or Windows rules. Based on the tags, it will fall under Linux regardless